docs/how-to/containerd-kata.md

```diff
@@ -312,10 +255,12 @@ To run a container with Kata Containers through the containerd command line, you
 
 ```bash
 $ sudo ctr image pull docker.io/library/busybox:latest
-$ sudo ctr run --runtime io.containerd.run.kata.v2 -t --rm docker.io/library/busybox:latest hello sh
+$ sudo ctr run --cni --runtime io.containerd.run.kata.v2 -t --rm docker.io/library/busybox:latest hello sh
```

Comment:
@amshinde testing this now, do you know if this should work with our default configurations?

Reply:
It would be great if you could update [`kata-manager`](https://github.com/kata-containers/kata-containers/blob/main/utils/README.md) to enable `--cni` too.

Reply:
Will do